test_conformance/conversions/conversions_data_info.h

```diff
@@ -125,8 +142,9 @@ struct DataInfoSpec : public DataInitBase
     }
 };
 
-template <typename InType, typename OutType>
-DataInfoSpec<InType, OutType>::DataInfoSpec(const DataInitInfo &agg)
+template <typename InType, typename OutType, bool InFP, bool OutFP>
+DataInfoSpec<InType, OutType, InFP, OutFP>::DataInfoSpec(
+    const DataInitInfo &agg)
     : DataInitBase(agg), mdv(0)
 {
     if (std::is_same<cl_float, OutType>::value)
```

Comment:
I think we need an if check here for the "half" OutType, to setup ranges properly for halfs.

Reply:
Done, I have added additional if-check for both `cl_half` and `cl_ushort`